package.json

```diff
@@ -41,24 +41,25 @@
     "broccoli-funnel": "^3.0.8",
     "ember-cli-babel": "^7.26.6",
     "ember-cli-htmlbars": "^5.7.1",
-    "ember-destroyable-polyfill": "^2.0.3"
+    "ember-destroyable-polyfill": "^2.0.3",
+    "ember-cli-typescript": "^4.2.1"
   },
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
+    "@embroider/test-setup": "^0.43.0",
     "@types/ember": "^3.16.5",
     "@types/ember-testing-helpers": "^0.0.4",
     "@types/rsvp": "^4.0.4",
     "@typescript-eslint/eslint-plugin": "^4.29.0",
     "@typescript-eslint/parser": "^4.29.0",
     "broccoli-merge-trees": "^4.2.0",
     "documentation": "^13.2.5",
-    "ember-auto-import": "^1.10.1",
+    "ember-auto-import": "^2.0.2",
     "ember-cli": "~3.27.0",
     "ember-cli-dependency-checker": "^3.2.0",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-shims": "^1.2.0",
     "ember-cli-test-loader": "^3.0.0",
-    "ember-cli-typescript": "^1.5.0",
```

Comment:
Moving to new versions of ember-cli-typescript continues the need to ship typescript, but provides embroider support.
There is another path, where we stop shipping typescript and rather do typescript as a pre-build step. Although I am not apposed to this, It's not clear its needed at this time.